client/src/Components/CommonComponent/StudentTable/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './index.css';
+
+const StudentTable = data => {
+  return (
+    <table className="parentTable">
```

Comment:
thanks for the changes! 👍 could you pls make another PR and update all of the CSS classnames to `studentTable`?